src/borg/testsuite/archiver.py

```diff
@@ -2154,6 +2154,41 @@ def test_verify_data(self):
     def test_verify_data_unencrypted(self):
         self._test_verify_data('--encryption', 'none')
 
+    def test_empty_repository(self):
+        with Repository(self.repository_location, exclusive=True) as repository:
+            for id_ in repository.list():
+                repository.delete(id_)
+            repository.commit()
+        self.cmd('check', self.repository_location, exit_code=1)
+
+    def test_attic013_acl_bug(self):
+        # Attic up to release 0.13 contained a bug where every item unintentionally received
+        # a b'acl'=None key-value pair.
+        # This bug can still live on in Borg repositories (through borg upgrade).
+        class Attic013Item:
+            as_dict = lambda: {
```

Comment:
./src/borg/testsuite/archiver.py:2169:13: E731 do not assign a lambda expression, use a def